sprocs/assessment_instances_select_log.sql

```diff
@@ -167,7 +167,7 @@ BEGIN
                     LEFT JOIN users AS u ON (u.user_id = gj.auth_user_id)
                 WHERE
                     iq.assessment_instance_id = ai_id
-                    AND gj.grading_method = 'Manual'
+                    AND gj.grading_method_manual = True
```

Comment:
Another example of where it looks like these boolean fields could live in the grading job column. We may have to unpack this logic a little to understand why these grading types are important here. 

I am not too sure what happens if an item has been internally and externally graded now, and how it relates to logging compared to if it was only an externally graded question, etc.

Reply:
Yeah, I'm a bit tentative on how to deal with joint logic now, mainly since before internally and externally graded questions have been completely disjoint. There's no support to internally and then externally grade--the external grading workflow, if it's run, will override the scores of the internally graded version of the question at this point in time. And if there's an internally graded portion, and then also an externally graded portion, we'll still have the two workflows stomping on each other, instead of creating a composite score.

After stepping back for a bit on this, what are your opinions on having an internal/external grading method column that forces the two to be disjoint, and then a second manual grading method column?